biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1651,8 +1661,33 @@ subroutine ConstrainRecruitNumber(csite,ccohort, bc_in)
           nmin = min(n, nmin) 
        endif
     end do
-    ccohort%n = min (ccohort%n, nmin) 
 
+    ! If the minimum number of plants is less than what was dictated by the
+    ! carbon-nitrogen-phosphorus availability, then we appy a reduction.
```

Comment:
isn't it the water availability that limits things in this logic, not the carbon-nitrogen-phosphorus availability?

Reply:
will clean that up, thanks